tf2_eigen/package.xml

```diff
@@ -1,20 +1,26 @@
 <?xml version="1.0"?>
```

Comment:
I'll open a PR against the ROS1 geometry2 to change it to format="2".  Once that is in, we can merge it up with this one.